plugins/in_kafka/in_kafka.c

```diff
@@ -243,6 +248,27 @@ static int in_kafka_init(struct flb_input_instance *ins,
         goto init_error;
     }
 
+    /* Set the kafka poll timeout dependend on wether we run in our own   
+       or in the main event thread.
+       a) run in main event thread:
+          -> minimize the delay we might create
+       b) run in our own thread:
+          -> optimize for throuput and relay on 'fetch.wait.max.ms'
+             which is set to 500 by default default. wa algin our 
+             timeout with what is set for 'fetch.wait.max.ms' */
```

Comment:
There could be existing typos:

```suggestion
    /* Set the kafka poll timeout depending on whether we run in our own
       or in the main event thread.
       a) run in main event thread:
          -> minimize the delay we might create
       b) run in our own thread:
          -> optimize for throughput and relay on 'fetch.wait.max.ms'
             which is set to 500 by default. we align our 
             timeout with what is set for 'fetch.wait.max.ms' */
```